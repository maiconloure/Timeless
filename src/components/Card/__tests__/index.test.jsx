import React from 'react';
import renderer from 'react-test-renderer';

import DefaultCard from '../index';

jest.mock('../../../utils/importAll', () => ({ icons: {} }));

const data = {
  checked: '',
  group: {
    color: '',
    number: 0,
  },
  users: {
    name: '',
    id: 0,
  },
  connected: 0,
  followers: { name: '', id: 0 },
  blocked: '',
  fastCard: '',
  title: '',
  time: {
    finish: '',
    start: '',
    done: '',
  },
  description: '',
  tags: ['', ''],
};

const image = {
  email: '',
  name: '',
  id: 0,
  image: '',
  about: '',
};

const removeCard = {
  selectedCard: {
    removeCard: '',
    fastCard: '',
    blockedCard: '',
  },
};

describe('Snapshot', () => {
  it('renderer', () => {
    const tree = renderer
      .create(
        <DefaultCard card={{ data }} user={{ image }} selectedCard={{ removeCard }} fastCard={{}} />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
