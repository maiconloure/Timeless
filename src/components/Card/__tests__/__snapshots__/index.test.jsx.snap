// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot renderer 1`] = `
<div
  className="sc-efHXLn ZtnDs"
  style={Object {}}
>
  <div
    className="sc-jNnnWF dbvHmM"
    draggable={false}
    style={
      Object {
        "touchAction": "none",
        "transform": "translateX(undefined) translateY(undefined) translateZ(0)",
        "userSelect": "none",
      }
    }
  >
    <div
      className="sc-cTJmaU GxEyK"
    >
      <div
        className="sc-dPaNSN ibmYNX"
        id="cardundefined"
        style={
          Object {
            "transform": "translateX(undefined) translateY(undefined) translateZ(0)",
          }
        }
      >
        <div
          className="sc-bBjRzc fUXUeS"
        >
          <span
            className="sc-jgPznn keREtS"
          >
            
          </span>
          <div
            className="sc-cOigif gcpPQW"
          >
            <div
              className="sc-ArjOu kjlBSZ"
            >
              <div
                className="sc-khIgXV ikuYwr"
                style={
                  Object {
                    "color": undefined,
                  }
                }
              />
              <div
                className="sc-khIgXV ikuYwr"
                style={
                  Object {
                    "color": undefined,
                  }
                }
              />
              <div>
                <figure
                  className="sc-gSYCTC cHhcay"
                >
                  <div
                    className="tooltip"
                  >
                    <img
                      alt="Have description"
                    />
                    <span
                      className="tooltiptext"
                    >
                      Possui descrição
                    </span>
                  </div>
                </figure>
              </div>
            </div>
            <div
              className="sc-hTRkEk"
            >
              <div
                className="tooltip"
              >
                <span />
                <span
                  className="tooltiptext"
                >
                  Tempo Estimado
                </span>
              </div>
            </div>
          </div>
          <section
            className="sc-laZMyp fkLiJr"
          >
            <div>
              <div>
                <h4
                  className="sc-iNiQeE iUmtWN"
                >
                  
                </h4>
              </div>
              <div>
                <p>
                  
                </p>
              </div>
            </div>
            <div>
              <input
                className="sc-jffIyK"
                defaultChecked=""
                type="checkbox"
              />
            </div>
          </section>
          <div
            className="sc-eJoaVz jtkqfq"
          >
            <figure
              className="sc-oefIU iMslNp"
            >
              <div
                className="tooltip"
              >
                <img
                  alt="user icon"
                  src={
                    Object {
                      "about": "",
                      "email": "",
                      "id": 0,
                      "image": "",
                      "name": "",
                    }
                  }
                />
                <span
                  className="tooltiptext"
                />
              </div>
            </figure>
            <div
              className="sc-hhIhEF sMkqH"
            >
              <div
                className="tooltip"
              >
                <span />
                <span
                  className="tooltiptext"
                >
                  Data de Entrega
                </span>
              </div>
            </div>
          </div>
        </div>
        <button
          className="sc-eirseW dcfdVp"
        >
          remover
        </button>
        
      </div>
    </div>
  </div>
</div>
`;
